build.gradle

```diff
@@ -22,95 +22,27 @@ ext {
 ext {
     // libs
     automatticAboutVersion = '1.4.0'
-    automatticRestVersion = '1.0.8'
     automatticTracksVersion = '5.1.0'
     gutenbergMobileVersion = 'v1.120.1'
     wordPressAztecVersion = 'v2.1.3'
     wordPressFluxCVersion = 'trunk-79f2bc35285748c715e00f2a6648ed6831632178'
     wordPressLoginVersion = '1.16.0'
-    wordPressPersistentEditTextVersion = '1.0.2'
     wordPressUtilsVersion = '3.14.0'
-    indexosMediaForMobileVersion = '43a9026f0973a2f0a74fa813132f6a16f7499c3a'
     gravatarVersion = '1.0.0'
 
-    // debug
-    flipperVersion = '0.245.0'
-    soLoaderVersion = '0.10.5'
-
     // main
-    androidInstallReferrerVersion = '2.2'
-    androidVolleyVersion = '1.2.1'
     androidxAnnotationVersion = '1.6.0'
-    androidxAppcompatVersion = '1.6.1'
-    androidxArchCoreVersion = '2.2.0'
-    androidxCameraVersion = '1.3.4'
     androidxComposeBomVersion = '2023.10.00'
     androidxComposeCompilerVersion = '1.5.9'
-    androidxComposeNavigationVersion = '2.7.6'
-    androidxCardviewVersion = '1.0.0'
-    androidxConstraintlayoutVersion = '2.1.4'
-    androidxConstraintlayoutComposeVersion = '1.0.1'
-    androidxCoreVersion = '1.10.0'
-    androidxActivityVersion = '1.8.0'
-    androidxFragmentVersion = '1.6.2'
-    androidxGridlayoutVersion = '1.0.0'
-    androidxLifecycleVersion = '2.6.2'
-    androidxPercentlayoutVersion = '1.0.0'
-    androidxPreferenceVersion = '1.2.0'
-    androidxRecyclerviewVersion = '1.3.0'
-    androidxSwipeToRefreshVersion = '1.1.0'
-    androidxViewpager2Version = '1.0.0'
-    androidxWorkManagerVersion = "2.9.0"
-    androidxWebkitVersion =  '1.11.0'
-    androidxComposeMaterial3Version = '1.1.1'
-    apacheCommonsTextVersion = '1.10.0'
-    coilComposeVersion = '2.4.0'
-    chrisbanesPhotoviewVersion = '2.3.0'
-    eventBusVersion = '3.3.1'
-    facebookShimmerVersion = '0.5.0'
+    androidxGridlayoutVersion = '1.0.0' // unused needs verify?
```

Comment:
Reviewer Note : Can you verify whether this ext is needed? Seem's like its no longer used. If yes can I safely remove it?

Reply:
Closing the conversations as its answered.